source/component/pasdoc.css

```diff
@@ -116,9 +169,34 @@ table.detail td.itemcode { width:100%; }
 td.itemname {white-space:nowrap; }
 td.itemunit {white-space:nowrap; }
 td.itemdesc { width:100%; }
+td.itemcode > code { color:#252525;}
+td.itemcode a {color: #000000;}
+td.itemcode a:hover {color: #f00;}
+td.itemcode code>a {color: #c91e0c;}
+td.itemcode code>a:hover {color: #f00;}
 
 div.nodescription { color:red; }
-dl.parameters dt { color:blue; }
+dl.parameters dt { 
+  display: inline-block;
+  color:blue; 
+}
+
+dl.parameters dd {
+  display: inline;
+  margin: 0;
+}
+
+dl.parameters dd:after{
+  display: block;
+  content: ';
+}
+
+code {
+  font-family: monospace;
+  font-size:1.2em;
+  font-style:  italic;
```

Comment:
I'll have to review some of these visual changes, I especially hesitate about this -- making code italic. I'll look at the resulting doc with new CSS later.

Reply:
font-italic Is not necessary like you can see on the screenshot

Reply:
Please remove the `font-italic` then.

Reply:
Removed